.github/workflows/ci.yml

```diff
@@ -40,12 +40,61 @@ jobs:
             *.cache-from=type=gha,scope=build-${{ matrix.target }}
             *.cache-to=type=gha,scope=build-${{ matrix.target }}
       -
-        name: Upload build artifacts
-        uses: actions/upload-artifact@v2
+        name: Upload artifacts
+        uses: actions/upload-artifact@v3
         with:
           name: ${{ matrix.target }}
-          path: ${{ env.BUNDLES_OUTPUT }}/${{ matrix.target }}-daemon/*
+          path: ${{ env.BUNDLES_OUTPUT }}
+          if-no-files-found: error
+          retention-days: 7
+
+  cross:
+    runs-on: ubuntu-20.04
+    strategy:
+      fail-fast: false
+      matrix:
+        platform:
+          - linux/amd64
+          - linux/arm/v5
+          - linux/arm/v6
+          - linux/arm/v7
+          - linux/arm64
+          - linux/ppc64le
+          - linux/s390x
+          - windows/amd64
```

Comment:
With https://github.com/moby/moby/pull/43475 merged, can this include `windows/arm64` as well?

Reply:
Yes I think it should work. I just tried with #43431 but not the current working tree.